articles/java-jdbc/index.md

```diff
@@ -0,0 +1,309 @@
+- `getStudents()` method returns a list of students from the database. ` resultSet = statement.executeQuery("select * from school.students");` executes the `SELECT` query that gets all the students from the database and stores the result in the `resultSet`. 
+- The `while (resultSet.next())` loops through the result set to get all the students' data returned.
+- The `getStudentByEmail(String email)` method returns a student from the database with the email password in the `SELECT` query.
+- The `saveStudent(Student student)` method inserts a new student passed to the method into the database.
+- The `deleteStudent(String email)` method deletes a student with the email passed to the `DELETE` query in the database.
+- This `updateStudent(Student student, String email)` method updates the student information of the student with the email passed to the `UPDATE` query.
+- The `close()` method closes the database connection.
+
+#### Main.java
+In this class, we create an instance of the `MySqlAccess` class and call its various methods to execute various database queries.
+```java
+import java.util.List;
+
+public class Main {
+    public static void main(String args[]) throws Exception {
+        MySqlAccess dao = new MySqlAccess();
+        //Creates a database connection
+        System.out.println("Creating Database Connection");
+        dao.databaseConnection();
+
+
+        //Insert 2 students into the database
+        System.out.println("Inserting Students to the database");
+        Student peter = new Student("Peter John", "peter@gmail.com", "Computer Science");
+        Student cathy = new Student("Catherine Williams", "catherine@gmail.com", "Electrical engineering");
+        dao.saveStudent(peter);
+        dao.saveStudent(cathy);
+
+        //Get all Students from the database
+        System.out.println("Getting all students from the database");
+        List<Student> students = dao.getStudents();
+        for (Student student : students) {
+            System.out.println(student.toString());
+        }
+
+        //Get Student by email address
+        System.out.println("Getting the students by email");
+        Student student = dao.getStudentByEmail("peter@gmail.com");
+        System.out.println(student.toString());
+
+        //Updating student name
+        System.out.println("updating student name");
+        //Update Catherine Williams name to Catherine William
+        cathy.setName("Catherine William");
+        dao.updateStudent(cathy, cathy.getEmail());
+
+        //delete student
+        System.out.println("Deleting a student from the database");
+        dao.deleteStudent("peter@gmail.com");
+
+        //close the database connection
+        System.out.println("Closing the database connection");
+        dao.close();
+
+    }
+}
+```
+
+### Creating the database
+- Start the  Apache and MySQL servers from the XAMPP control panel.
+- On the `PHPMyAdmin` site on your browser create a database with the name `school`.
```

Comment:
How can I access PHPMyAdmin. Inform users of the link `localhost/phpmyadmin/` This will make it much friendlier for beginners